README.md

```diff
@@ -1,34 +1,40 @@
 # Coordinating HPC Carpentry
 
-This repository serves as a centralizing location for documents / notes / information re: HPC Carpentry 
-[HPC Carpentry organization](https://github.com/hpc-carpentry) that isn't contained on the general website. 
+This repository serves as a centralizing location for documents, notes, and information regarding
+the [HPC Carpentry organization](https://github.com/hpc-carpentry) that isn't contained on the
+[general website](https://hpc-carpentry.github.io/).
 
 ## Current status of HPC Carpentry
 
-"Beta" lessons are available in the [HPC Carpentry Github organization](https://github.com/hpc-carpentry).
+"Alpha" lessons are available in the [HPC Carpentry organization](https://github.com/hpc-carpentry).
```

Comment:
In reference to https://github.com/hpc-carpentry/hpc-intro/issues/174, I think it is useful to make it clear everywhere, that we consider the material in alpha stage for now.